src/components/CommentBubble/index.tsx

```diff
@@ -0,0 +1,21 @@
+import "./styles.scss";
+import { CommentProps } from "../../data/interfaces";
+
+const CommentBubble = ({ comment }: CommentProps) => (
+  <>
```

Comment:
Component name + Props for interface please,so ** CommentBubbleProps** should be if CommentBubble is component name

Reply:
Why CommentBubble? it's not comment neither bubble, it's more ChatMessage